circleguard/investigator.py

```diff
@@ -264,6 +268,20 @@ def _filter_hits(hitobjs, keypress_times, OD):
 
         return array
 
+    @staticmethod
+    def _remove_low_frametimes(frametimes, limit):
+        cnt = 0
+        lowframes = []
+        for i, curr_t in enumerate(frametimes):
```

Comment:
what's with the brevity? No need to abbreviate things like this. Use `frametime` instead of `curr_t`, `num_low_frametimes` instead of `cnt`, and `low_frametimes` instead of `lowframes`